Code/Tools/FBuild/FBuildCore/Graph/ObjectNode.cpp

```diff
@@ -1454,6 +1454,39 @@ bool ObjectNode::BuildArgs( const Job * job, Args & fullArgs, Pass pass, bool us
                 }
             }
         }
+	
+	//remove static analyzer from clang preprocessor
+	if ( pass == PASS_PREPROCESSOR_ONLY )
+	{
+	   if ( isClang )
+	   {
+	      if ( StripTokenWithArg( "--analyze", token, i ) )
+	      {
+		  continue;
+	      }
+
+	      if ( StripTokenWithArg( "-Xanalyzer", token, i ) )
+	      {
+		  continue;
+	      }
+
+	      if ( StripTokenWithArg( "-analyzer-output", token, i ) )
+	      {
+		  continue;
+	      }
+
+	      if ( StripTokenWithArg( "-analyzer-config", token, i ) )
+	      {
+		  continue;
+	      }
+
+	      if ( StripTokenWithArg( "-analyzer-checker", token, i ) )
+	      {
+		  continue;
+	      }
+	   }
+	}
```

Comment:
It looks like your change mixes tabs and spaces and uses inconsistent tab widths.  It would simplify integration if you can consistently use 4-space indentation with no tab characters (all spaces).